Description: This patch claims to fix a bug in boinc-launching from different directories than /var/lib/boinc-client
Author: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
---
Origin: <vendor|upstream|other>, <url of original patch>
Bug-Debian: http://bugs.debian.org/646583
Bug-Ubuntu: https://launchpad.net/bugs/347651
Forwarded: not-needed
Last-Update: <2013-02-03>

--- boinc-7.0.47+dfsg.orig/lib/common_defs.h
+++ boinc-7.0.47+dfsg/lib/common_defs.h
@@ -224,7 +224,7 @@ struct VERSION_INFO {
 #define CLIENT_AUTH_FILENAME        "client_auth.xml"
 #define LOCK_FILE_NAME              "lockfile"
 #define GRAPHICS_APP_FILENAME       "graphics_app"
-#define GUI_RPC_PASSWD_FILE         "gui_rpc_auth.cfg"
+#define GUI_RPC_PASSWD_FILE         "/var/lib/boinc-client/gui_rpc_auth.cfg"
 #define SS_CONFIG_FILE              "ss_config.xml"
 
 #ifdef _WIN32
